### PR TITLE
[keras/layers/preprocessing] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/layers/preprocessing/category_encoding.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ class CategoryEncoding(base_layer.Layer):
         inputs to the layer must integers in the range `0 <= value <
         num_tokens`, or an error will be thrown.
       output_mode: Specification for the output of the layer.
-        Defaults to `"multi_hot"`. Values can be `"one_hot"`, `"multi_hot"` or
+        Values can be `"one_hot"`, `"multi_hot"` or
```
Comment (qlzh727) [inline keras/layers/preprocessing/category_encoding.py:93]: I think we should remove the double quote here, and just make it `one_hot`. Same for all the changes in this PR.

File: keras/layers/preprocessing/hashing.py [hunk 2]
```diff
@@ -109,17 +109,16 @@ class Hashing(base_layer.Layer):
         bin, so the effective number of bins is `(num_bins - 1)` if `mask_value`
         is set.
       mask_value: A value that represents masked inputs, which are mapped to
-        index 0. Defaults to None, meaning no mask term will be added and the
-        hashing will start at index 0.
+        index 0. None means no mask term will be added and the
```
Comment (qlzh727) [inline keras/layers/preprocessing/hashing.py:112]: Please quote the None here.

File: keras/layers/preprocessing/hashing.py [hunk 3]
```diff
@@ -109,17 +109,16 @@ class Hashing(base_layer.Layer):
...
       salt: A single unsigned integer or None.
         If passed, the hash function used will be SipHash64, with these values
         used as an additional input (known as a "salt" in cryptography).
-        These should be non-zero. Defaults to `None` (in that
-        case, the FarmHash64 hash function is used). It also supports
-        tuple/list of 2 unsigned integer numbers, see reference paper for
-        details.
-      output_mode: Specification for the output of the layer. Defaults to
-        `"int"`.  Values can be `"int"`, `"one_hot"`, `"multi_hot"`, or
+        These should be non-zero. If None, uses the FarmHash64 hash function.
```
Comment (qlzh727) [inline keras/layers/preprocessing/hashing.py:117]: Same here. Please quote the None.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/layers/preprocessing/category_encoding.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/layers/preprocessing/category_encoding.py:93]: `Union["one_hot"`, "multi_hot","count"]` is no longer the type?

Comment (SamuelMarks) [reply keras/layers/preprocessing/category_encoding.py:93]: `Union["one_hot", "multi_hot","count"]` is no longer the type?

File: keras/layers/preprocessing/hashing.py (hunk 2 above)
Comment (qlzh727) [reply keras/layers/preprocessing/hashing.py:112]: Please addres.

File: keras/layers/preprocessing/hashing.py (hunk 3 above)
Comment (qlzh727) [reply keras/layers/preprocessing/hashing.py:117]: Same.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/layers/preprocessing/hashing.py (hunk 2 above)
Comment (SamuelMarks) [reply keras/layers/preprocessing/hashing.py:112]: Done

File: keras/layers/preprocessing/hashing.py (hunk 3 above)
Comment (SamuelMarks) [reply keras/layers/preprocessing/hashing.py:117]: Done

**Pull request merged**